docs/docs/ias.md

```diff
@@ -1,41 +1,49 @@
+    DCAP attestation protocol.
 
-How do you know a remote computer is fully up to date with the latest security patches and isn't physically compromised?
-This is the question Intel's Attestation Service (IAS) exists to answer. IAS is an online service that evaluates
-evidence from an enclave and returns:
+Intel's Attestation Service (IAS) helps to verify that a remote computer isn't physically compromised and has the 
+latest security patches.
 
-1. The time at which the assessment was made
-2. Whether the system is considered secure or whether it needs:
+IAS evaluates evidence from an enclave and returns:
+
+1. The time at which the IAS made the assessment.
+2. If the system is secure or if it needs:
    1. Software upgrades
-   2. Operating system / microcode upgrades
+   2. OS or microcode upgrades
    3. BIOS configuration changes
-3. The CVE IDs of any active security advisories against the remote system
+3. The [CVE ID](https://en.wikipedia.org/wiki/Common_Vulnerabilities_and_Exposures#CVE_identifiers)s of relevant
+security advisories.
 
-... along with various other pieces of data. The response is signed by Intel. Because it's signed it can be hosted and
-passed around by anyone, thus when an enclave is started in debug or production mode IAS is contacted and an attestation
-calculated. It's a part of a serialized [`EnclaveInstanceInfo`](api/-conclave%20-core/com.r3.conclave.common/-enclave-instance-info/index.html) object and can be passed to clients from the server. This
-means clients don't need to contact Intel's servers themselves and thus don't need authorisation to use IAS.
+You can send and use the response anywhere, as Intel has signed it. When an enclave starts in debug or release mode, 
+it contacts the IAS to calculate the attestation. This attestation is a part of a serialized
+[`EnclaveInstanceInfo`](api/-conclave%20-core/com.r3.conclave.common/-enclave-instance-info/index.html) object and 
+can be passed to clients from the server. This means clients don't need authorization to use the IAS.
 
 ## Getting access
 
-IAS requires an authorised key to access it. This is distinct from the enclave signing key. To obtain an IAS access
-key: 
+You need an authorized key to access the IAS. This key is different from the enclave signing key.
+
+To obtain an IAS access key: 
 
-1. Sign the commercial use agreement [as part of getting a whitelisted enclave signing key](signing.md). You can run your application in simulation or debug mode without this, but try to do it well in advance of when you will need to run in release mode, as the application process can be lengthy.
-2. Get an Intel developer zone account and sign in [on the IAS API site](https://api.portal.trustedservices.intel.com/EPID-attestation).
-3. Subscribe to development and production access for "EPID Name Base Mode (Linkable Quotes)".
+1. Sign the commercial use agreement [as part of getting an enclave signing key](signing.md). You need to do this 
```

Comment:
I would put the link on `Sign the commercial use agreement`

Reply:
Done.